Sources/StreamChatUI/ChatChannelList/ChatChannelListErrorView.swift

```diff
@@ -0,0 +1,54 @@
+//
+//  ChatChannelListErrorView.swift
+//  StreamChat
+//
+//  Created by Hugo Bernal on 12/07/22.
+//  Copyright © 2022 Stream.io Inc. All rights reserved.
+//
+
+import UIKit
+
+open class ChatChannelListErrorView: _View, ThemeProvider {
+    
+    /// Container which holds the elements on the error banner.
+    open private(set) lazy var container: ContainerStackView = ContainerStackView().withoutAutoresizingMaskConstraints
+    /// Label describing there has been an error.
+    open private(set) lazy var titleLabel: UILabel = UILabel().withBidirectionalLanguagesSupport.withoutAutoresizingMaskConstraints
+    /// Retry button which is located at the trailing of the view.
+    open private(set) lazy var retryButton: UIButton = UIButton().withoutAutoresizingMaskConstraints
+    /// Spacing view so the button has some spacing with bidirectional language support.
+    open private(set) lazy var spacer: UIView = UIView().withoutAutoresizingMaskConstraints
+    
+    open var refreshButtonAction: (() -> Void)?
+    
+    override open func setUp() {
+        super.setUp()
+
+        titleLabel.text = L10n.ChannelList.Error.message
+        retryButton.addTarget(self, action: #selector(didTapRetryButton), for: .touchUpInside)
+    }
+
+    override open func setUpAppearance() {
+        super.setUpAppearance()
+        retryButton.setImage(appearance.images.restart, for: .normal)
+        backgroundColor = appearance.colorPalette.textLowEmphasis
+        titleLabel.textColor = appearance.colorPalette.staticColorText
+        retryButton.tintColor = appearance.colorPalette.staticColorText
+    }
+
+    override open func setUpLayout() {
+        super.setUpLayout()
+
+        directionalLayoutMargins = .init(top: 18, leading: 16, bottom: 18, trailing: 16)
```

Comment:
Why 18? I think we should use 16 everywhere.

Reply:
That's how is in the design specs (https://www.figma.com/file/ekifwChR9tR7zRJg1QEzSM/Chat-Design-Kit---Current-version?node-id=24694%3A348732) so I used those values for margins, but it does make sense to 16 everywhere. Should I change it then?

Reply:
Yes, please do. Might be a mistake in the designs - we keep standard paddings (8, 16, etc).

Reply:
Done